types/office-js/index.d.ts

```diff
@@ -1974,6 +1974,14 @@ declare namespace Office {
          * Triggers when a customXmlPart node was replaced.
          */
         NodeReplaced,
+        /**
+         * Triggers when the OfficeTheme was changed in Outlook.
```

Comment:
"was changed" -> "changes"

Reply:
Per our discussion, updated to "is changed"